app/models/manageiq/providers/azure/cloud_manager/event_parser.rb

```diff
@@ -0,0 +1,18 @@
+module ManageIQ::Providers::Azure::CloudManager::EventParser
+  extend ManageIQ::Providers::Azure::EventCatcherMixin
+
+  def self.event_to_hash(event, ems_id)
```

Comment:
@Fryguy this is the way how all the other `EventParsers` do it. So I'd continue doing this.
But I have a rework of this cooking in #7104